opencti-platform/opencti-front/src/private/components/common/containers/ContainerSuggestedMappingContent.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { FunctionComponent, useState } from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import { graphql, useFragment } from 'react-relay';
+import Grid from '@mui/material/Grid';
+import StixCoreObjectMappableContent from '@components/common/stix_core_objects/StixCoreObjectMappableContent';
+import Paper from '@mui/material/Paper';
+import { Link, useNavigate } from 'react-router-dom';
+import { containerContentFragment, contentMutationFieldPatch } from '@components/common/containers/ContainerContent';
+import ContainerStixCoreObjectsSuggestedMapping, { containerStixCoreObjectsSuggestedMappingQuery } from '@components/common/containers/ContainerStixCoreObjectsSuggestedMapping';
+import {
+  ContainerSuggestedMappingContentAddSuggestedMappingRelationsMutation,
+} from '@components/common/containers/__generated__/ContainerSuggestedMappingContentAddSuggestedMappingRelationsMutation.graphql';
+import { ContainerContentFieldPatchMutation } from '@components/common/containers/__generated__/ContainerContentFieldPatchMutation.graphql';
+import { ContainerStixCoreObjectsSuggestedMappingQuery$data } from '@components/common/containers/__generated__/ContainerStixCoreObjectsSuggestedMappingQuery.graphql';
+import { ContainerContent_container$key } from '@components/common/containers/__generated__/ContainerContent_container.graphql';
+import { ContainerContentQuery$data } from '@components/common/containers/__generated__/ContainerContentQuery.graphql';
+import { useFormatter } from '../../../../components/i18n';
+import { MESSAGING$, QueryRenderer } from '../../../../relay/environment';
+import useApiMutation from '../../../../utils/hooks/useApiMutation';
+import { decodeMappingData, encodeMappingData } from '../../../../utils/Graph';
+import { resolveLink } from '../../../../utils/Entity';
+import { ContainerSuggestedMappingContentAskSuggestedMappingMutation } from './__generated__/ContainerSuggestedMappingContentAskSuggestedMappingMutation.graphql';
+
+const addSuggestedMappingRelationsMutation = graphql`
+    mutation ContainerSuggestedMappingContentAddSuggestedMappingRelationsMutation(
+        $id: ID!
+        $input: StixRefRelationshipsAddInput!
+        $commitMessage: String
+        $references: [String]
+    ) {
+        stixCoreObjectEdit(id: $id) {
+            relationsAdd(
```

Comment:
use FieldPatch along with mappingContent to send only one mutation instead of two